src/viewer.js

```diff
@@ -116,6 +116,29 @@ module.exports = class Viewer {
     this.gridHelper = null;
     this.axesHelper = null;
 
+    // https://stackoverflow.com/questions/16226693/three-js-show-world-coordinate-axes-in-corner-of-scene 
+
+    this.axisDiv = document.createElement('div');
+    this.el.appendChild( this.axisDiv );
+    this.axisDiv.classList.add('axis');
+
+    this.axisScene = new THREE.Scene();
+    this.axisCamera = new THREE.PerspectiveCamera( fov, this.axisDiv.clientWidth / this.axisDiv.clientHeight, 0.01, 10000 );
+    this.axisScene.add( this.axisCamera );
+
+    this.axisRenderer = new THREE.WebGLRenderer( { alpha: true } );
+    this.axisRenderer.physicallyCorrectLights = true;
+    this.axisRenderer.gammaOutput = true;
+    this.axisRenderer.gammaFactor = 2.2;
```

Comment:
No need for these three settings (`physicallyCorrectLights`, `gammaOutput`, and `gammaFactor`). The axes are unlit, and color correction + lighting shouldn't be necessary.

Reply:
Done.